sdcm/remote/kubernetes_cmd_runner.py

```diff
@@ -511,7 +519,8 @@ def run(self, command, **kwargs):
         environ = self.context.config.k8s_environ
         environ["K8S_POD_COMMAND"] = command
         environ["K8S_POD_NAME"] = pod_name
-        environ["DOCKER_IMAGE_WITH_TAG"] = self._get_docker_image(command)
+        if not environ.get("DOCKER_IMAGE_WITH_TAG"):
```

Comment:
In which can we don't want to over run it ?

Reply:
DNS pod is special and it doesn't use any of the loader images.
This logic was designed for loader images.
So, DNS pod will provide env var directly with it's special DNS image and all other loader pods will get it automatically based on the provided cmd.